crates/typst-library/src/introspection/counter.rs

```diff
@@ -436,29 +422,24 @@ impl Counter {
         self.at_loc(engine, loc)
     }
 
-    /// Displays the current value of the counter with a numbering and returns
-    /// the formatted output.
+    /// 番号付けされたカウンターの現在の値の表示および戻り値としての整形された出力。
     #[func(contextual)]
     pub fn display(
         self,
         engine: &mut Engine,
         context: Tracked<Context>,
         span: Span,
-        /// A [numbering pattern or a function]($numbering), which specifies how
-        /// to display the counter. If given a function, that function receives
-        /// each number of the counter as a separate argument. If the amount of
-        /// numbers varies, e.g. for the heading argument, you can use an
-        /// [argument sink]($arguments).
+        /// カウンターをどのように表示するか指定する[番号付けのパターンまたは関数]($numbering)。
+        /// 関数が与えられた場合、カウンターの各数値が別々の引数として関数に渡されます。
+        /// 見出しの番号付けを関数指定するときなどの、引数の数値の数が変化する場合は[引数シンク]($arguments)が使用できます。
         ///
-        /// If this is omitted or set to `{auto}`, displays the counter with the
-        /// numbering style for the counted element or with the pattern
-        /// `{"1.1"}` if no such style exists.
+        /// 省略されるか`{auto}`に設定された場合、カウントする要素に設定された番号付け形式を用いてカウンターを表示します。
+        /// そのようなスタイルが存在しない場合は`{"1.1"}`というパターン指定でカウンターが表示されます。
         #[default]
         numbering: Smart<Numbering>,
-        /// If enabled, displays the current and final top-level count together.
-        /// Both can be styled through a single numbering pattern. This is used
-        /// by the page numbering property to display the current and total
-        /// number of pages when a pattern like `{"1 / 1"}` is given.
+        /// 有効化された場合、トップレベルのカウンターの現在値と最終値を一緒に表示します。
+        /// 両者のスタイル設定は1つの番号付けパターンで指定できます。
+        /// これは、`{"1 / 1"}`のようなパターンが与えられたときに、現在のページ番号と全ページ数を表示するためにページの番号付けプロパティによって使用されます。
```

Comment:
「現在のページ番号と全ページ数を表示するために」「ページの番号付けプロパティによって」「使用されます。」の順番ですが、「現在のページ番号と総ページ数を表示するために」と「使用されます。」がくっついた方が、個人的には読みやすく思います。

非常に細かいコメントですが、「全ページ数」より「総ページ数」としたほうが、総計された数字というニュアンスが出せると思います。

```
        /// これは、`{"1 / 1"}`のようなパターンが与えられたときに、ページの番号付けプロパティによって、現在のページ番号と総ページ数を表示するために使用されます。
```

もう少し大胆に変えると以下のような意味かと思います。

```suggestion
        /// これは、ページのnumberingプロパティで`{"1 / 1"}`のようなパターンが与えられたときに、現在のページ番号と総ページ数を表示するために使用されます。
```

numberingはプロパティなのであえて、和訳せずに置きました。